tools/reviewer/web/tslint.json

```diff
@@ -20,7 +20,7 @@
       "check-preblock"
     ],
     "typedef": [
-      true,
+      false, // false, until all typedef issues aren't fix
```

Comment:
Did you mean this? s/aren't fix/are fixed
Is this a `TODO`? If so please mark it as such.

Reply:
Done